py/client/pydeephaven/session.py

```diff
@@ -127,13 +154,63 @@ def get_ticket(self):
 
             return self._last_ticket
 
+    def sync_fields(self, repeating: bool):
+        """ Check for fields that have been added/deleted by other sessions and add them to the local list
+
+        This will start a new background thread when `repeating=True`.
```

Comment:
It should be noted - when repeating, only the *first* response will modify global "scope" - future responses against global "scope" will *not* be handled.

Reply:
Since the issue with script response/listfields response conflicts is fixed, can the repeating sync_fields be switched back to handling all changes?

Reply:
Yes, I think that's correct. But my other comment is still relevant - we want run_script changes to be visible upon return.